lib/_stream_wrap.js

```diff
@@ -10,9 +10,11 @@ function StreamWrap(stream) {
 
   this.stream = stream;
 
+  this.queue = null;
```

Comment:
@thefourtheye there is no promise of FIFO here, and I like linked lists pretty much :)